client/src/templates/Challenges/components/help-modal.tsx

```diff
@@ -45,6 +45,40 @@ const generateSearchLink = (title: string, block: string) => {
   return search;
 };
 
+interface CheckbockProps {
+  name: string;
+  i18nkey: string;
+  onChange: (event: React.ChangeEvent<HTMLInputElement>) => void;
+  value: boolean;
+  href: string;
+  t: (text: string) => string;
+}
+
+function CheckboxHelpModal({
+  name,
+  i18nkey,
+  onChange,
+  value,
+  href,
+  t
+}: CheckbockProps) {
+  return (
+    <div style={{ display: 'flex', flexDirection: 'row' }}>
+      <input name={name} type='checkbox' onChange={onChange} checked={value} />
+      <Trans i18nKey={i18nkey}>
+        <a
+          href={href}
+          rel='noopener noreferrer'
+          target='_blank'
+          title={t(i18nkey)}
+        >
+          {t(i18nkey)}
```

Comment:
```suggestion
          {t(i18nkey)} <span className='sr-only'>{t('aria.opens-new-window')}
```

I think we need to let screen reader users know that these links open a new tab/window.